.github/workflows/test.yml

```diff
@@ -54,6 +54,10 @@ jobs:
         run: |
           pipenv install -r dev-requirements.txt
 
+      - name: Install doc dependencies
+        run: |
+          pipenv install -r doc-requirements.txt
+
```

Comment:
it might make sense to have a separate workflow for building the docs (like `docs.yml` or something). Since it would be useful to know if say the tests are failing but the docs are building correctly in a given PR.

Reply:
As you prefer ; it would make sense to have one.